src/common/messages/floor/ElevatorFloorRequest.java

```diff
@@ -0,0 +1,25 @@
+package common.messages.floor;
+
+import common.Direction;
+import common.messages.ElevatorJobMessage;
+import common.messages.Message;
+import common.messages.MessageType;
+
+public class ElevatorFloorRequest extends Message implements ElevatorJobMessage {
```

Comment:
Like I mentioned, the classes are super similar.

Consider renaming to FloorPassengerElevatorRequest.

Reply:
In theory, you could make ElevatorJobMessage extend Message and implement the required functions there.

That way you do not have repeat yourself.